method/message/utility/base64String.json

```diff
@@ -0,0 +1,6 @@
+{
+	"description": "Match string in Base64",
+	"type": "string",
+	"contentEncoding": "base64",
+	"$comment": "Note: JsonSchema doesn't check that the string is some valid Base64 string"
```

Comment:
It seems possible?
https://stackoverflow.com/questions/475074/regex-to-parse-or-validate-base64-data

Reply:
It looks still hard. Worse case, we can check it after the JSON schema. We have to verify signatures and hashs anyway.

Reply:
Agreed, you can use an ugly regex ( e.g. `^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$` ) and parse the content once you decode it :)

Reply:
Forget my last message... and read the docs!
https://json-schema.org/understanding-json-schema/reference/non_json_data.html

Reply:
@pierluca I read this page, this is what I did (set `contentEncoding` to `base64`). But I found that all strings were accepted (even non-base64 strings) through testing.

I also tried specifying `binaryEncoding` encoding instead of `contentEncoding` (keyword used by JSON Hyper-Schema Draft #4) but the result is the same